indexer/postgres/CHANGELOG.md

```diff
@@ -35,3 +35,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 # Changelog
 
 ## [Unreleased]
+
+## [v0.1.0](https://github.com/cosmos/cosmos-sdk/releases/tag/indexer/postgres/v0.1.0)
```

Comment:
does this need to be included?

Reply:
Theoretically it didn't but it saved me a PR to do it here directly.